docs/ble_hid.md

```diff
@@ -2,11 +2,9 @@
 Bluetooth connections help clean up the wire mess!
 
 ## CircuitPython
-If not running KMKPython, this does require the adafruit_ble library from Adafruit.
-This can be downloaded
-[here](https://github.com/adafruit/Adafruit_CircuitPython_BLE/tree/master/adafruit_ble).
-It is part of the [Adafruit CircuitPython Bundle](https://github.com/adafruit/Adafruit_CircuitPython_Bundle).
-Simply put this in the "root" of your CircuitPython device. If unsure, it's the folder with main.py in it, and should be the first folder you see when you open the device.
+If not running KMKPython, this does require the [adafruit_ble library from Adafruit](https://github.com/adafruit/Adafruit_CircuitPython_BLE/tree/master/adafruit_ble).
+It's part of the Adafruit CircuitPython Bundle, which can be downloaded [here](https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/tag/20220715).
+Get the bundle version that corresponds to the CircuitPython version installed on your device. Unzip the bundle and open the "lib" folder to locate the "adafruit_ble" library. Selectively copy that library (the entire "adafruit_ble" folder) and place it into the "lib" folder located at the root of your CIRCUITPY device. If unsure, the root is the main device storage folder where your main.py (or code.py) file is located. The "lib" folder should have been created when you installed CircuitPython. If not, just create one to place the ble library.
```

Comment:
Why doesn't the device root work? Is there some specific code somewhere that explicitly sources from _lib_?

Reply:
Our standard convention seems to be on the root of the drive. If that library somehow only magically works only under `/lib` then an exception can be made, however that shouldn't be the case last I worked on BLE